src/app/components/sidenav/sidenav.component.html

```diff
@@ -1,11 +1,11 @@
 <ng-container *ngIf="isSmallViewPort; else largeViewPort">
-  <forge-drawer (window:resize)="adjustDrawer()" (forge-drawer-close)="closeSidenav()" type="modal" open="false" [type]="drawerType" [open]="open">
+  <forge-modal-drawer (window:resize)="adjustDrawer()" (forge-drawer-close)="closeSidenav()" type="modal" open="false" [open]="open">
     <ng-container *ngTemplateOutlet="sidenavContent"></ng-container>
-  </forge-drawer>
+  </forge-modal-drawer>
 </ng-container>
 
 <ng-template #largeViewPort>
-  <forge-drawer (window:resize)="adjustDrawer()" (forge-drawer-close)="closeSidenav()" [type]="drawerType" [open]="open">
+  <forge-drawer (window:resize)="adjustDrawer()" (forge-drawer-close)="closeSidenav()" [open]="open">
```

Comment:
Some invalid markup caught in this file by removing CUSTOM_ELEMENTS_SCHEMA in favor of the proxy components.  The `forge-drawer [type]` was never migrated with Forge 2.0 so would never have been modal.  Also, `<forge-list-divider>` below doesn't actually exist.